src/Components/Contacts.js

```diff
@@ -0,0 +1,15 @@
+import ContactUserCard from "./ContactUserCard"
+
+export default function Contacts() {
+    ContactUserCard.forEach((contact) => {
+        return (
```

Comment:
Which function does this return from? (hint: the Contacts function has no return statement)